.drone.yml

```diff
@@ -364,6 +364,26 @@ steps:
     event:
     - push
 
+- name: push-master-3rd-party-manifest
+  image: plugins/docker
+  commands:
+    - echo $${DOCKER_PASSWORD} | docker login -u "$${DOCKER_USERNAME}" --password-stdin
+    - ./scripts/publish-manifests.sh
+  environment:
+    DOCKER_USERNAME:
+      from_secret: docker_username
+    DOCKER_PASSWORD:
+      from_secret: docker_password
+  privileged: true
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  when:
+    branch:
+    - master
```

Comment:
This should also only be done on tag.

Reply:
@alena1108 do we need these on RCs or only on final builds?

Reply:
I would assume we would want it for RCs and alphas, because otherwise they might fail to bring up those services.

Reply:
So pretty much any tag?